strings/src/main/res/values/strings.xml

```diff
@@ -1111,6 +1111,8 @@
 
     <!-- Error shown when a user tries to open a form that's in a different project -->
     <string name="wrong_project_selected_for_form">To open this form, you must first open ODK Collect and switch to the project that contains it.</string>
+    <!-- Error shown when a user tries to open a form when there are no projects -->
+    <string name="no_projects_detected">No projects detected.</string>
```

Comment:
This is a weird one because it means the shortcut/URI will never work right? If there are no projects, they'd need to add one and the form would end up with a different ID (probably).

Reply:
Yes it would work in case of demo projects but other projects will have unique ids.

Reply:
Ok how about: 

```
"ODK Collect has not been configured. Try to open ODK Collect and configure it.\n\nIf you tapped on a shortcut, you might need to recreate it after configuring ODK Collect."
```

Reply:
I agree it's better, done.